src/main/java/graphql/analysis/MaxQueryComplexityInstrumentation.java

```diff
@@ -76,41 +81,52 @@ public MaxQueryComplexityInstrumentation(int maxComplexity, FieldComplexityCalcu
         this.maxQueryComplexityExceededFunction = maxQueryComplexityExceededFunction;
     }
 
+    @Override
+    public InstrumentationState createState(InstrumentationCreateStateParameters parameters) {
+        return new State();
+    }
```

Comment:
Only needed so we can capture `InstrumentationValidationParameters` parameters to maintain API